### PR TITLE
Added collaboration hook description

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu/pull/2201

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@danrot i have removed the autoStart flag from this PR

File: bundles/admin/javascript-hooks/collaboration.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Collaboration
+=============
+
+The collaboration-hook initializes collabation-component. This component i able
```
Comment (danrot) [inline bundles/admin/javascript-hooks/collaboration.rst]: `is able`

File: bundles/admin/javascript-hooks/collaboration.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        },
+
+        initialize: function() {
+            render();
+        },
+
+        ...
+    }
+
+This basic component initializes the collaboration and get a warning when a user
```
Comment (danrot) [inline bundles/admin/javascript-hooks/collaboration.rst]: `and shows a warning`

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**